### DISCUSSION TITLE
카카오 잔여백신 지도 오류

### DISCUSSION THREAD

**Discussion by Min0317** [changdoc/naver-vaccine-macro#191] [Q&A] [answered]
카카오 잔여백신 지도에 접속하여 잔여백신 핀을 누를 때 "지원되지 않는 환경입니다" 라는 메시지가 표출됩니다. 이에 대한 이유가 궁금합니다.

추가로, 네이버 지도를 통한 예약과 카카오 지도를 통한 예약을 동시어 병행해도 괜찮을까요?

**Comment by changdoc** [comment, marked as answer]
카카오 잔여백신 맵 자체에 카카오톡에서만 동작하게 하기 위해서 마우스 이벤트가 발생하면 "지원되지 않는 환경입니다" 오류 창을 띄우도록 되어 있는 것 같습니다.

네 동일한 지역으로 매크로 데이터 설정 한 뒤 동시에 병행하면 확률이 더 높았던 것 같습니다.

**Reply by Min0317** [reply]
감사합니다.
